otter/test/integration/lib.py

```diff
@@ -177,6 +179,127 @@ def record_results(resp):
         )
 
 
+@attributes([
+    Attribute('scale_by', instance_of=int),
+    Attribute('scaling_group', instance_of=ScalingGroup),
+])
+class ScalingPolicy(object):
+    """ScalingPolicy class instances represent individual policies which your
+    integration tests can execute at their convenience.
+
+    :param int scale_by: The number of servers to scale up (positive) or down
+        (negative) by.  Cannot be zero, lest an API-generated error occur.
+    :param ScalingGroup scaling_group: The scaling group to which this policy
+        applies.
+    """
+
+    def __init__(self):
+        self.policy = [{
+            "name": "integration-test-policy",
+            "cooldown": 0,
+            "type": "webhook",
+            "change": self.scale_by
```

Comment:
Yes.  YAGNI.  When integration tests covering those features are written, we can add them in at that time.